packages/l/libusb/xmake.lua

```diff
@@ -5,18 +5,41 @@ package("libusb")
 
     set_urls("https://github.com/libusb/libusb/archive/$(version).tar.gz",
              "https://github.com/libusb/libusb.git")
+    add_versions("v1.0.24", "b7724c272dfc5713dce88ff717efd60f021ca5b7c8e30f08ebb2c42d2eea08ae")
     add_versions("v1.0.23", "02620708c4eea7e736240a623b0b156650c39bfa93a14bcfa5f3e05270313eba")
 
     if not is_host("windows") then
         add_deps("autoconf", "automake", "libtool", "pkg-config")
     end
 
-    add_includedirs("include/libusb-1.0")
+    on_install("windows", function (package)
+        os.cd("msvc")
+
+        local configs = {"libusb_2019.sln"}
+        local arch = package:is_arch("x86") and "Win32" or "x64"
+        local mode = package:debug() and "Debug" or "Release"
+
+        table.insert(configs, "/property:Configuration=" .. mode)
+        table.insert(configs, "/property:Platform=" .. arch)
+        --table.insert(configs, "-t:" .. (package:config("shared") and "libusb-1_0%20_dll_" or "libusb-1_0%20_static_"))
```

Comment:
Does it not work? `libusb_dll_2019.vcxproj` `libusb_static_2019.vcxproj`

libusb_dll_2019? libusb_static_2019?

Reply:
I don't recall the error but it didn't work